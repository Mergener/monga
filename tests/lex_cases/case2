var mee function

while 4424 
	1.5    <=