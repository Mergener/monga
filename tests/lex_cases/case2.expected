var
identifier mee
function
while
lit_int 4424
lit_float 1.500
op_le